deployment/ansible/roles/dcld/templates/dcld.service.j2

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=DC Ledger service
+After=network.target
+StartLimitIntervalSec=0
+
+[Service]
+User={{ dcld.user }}
+Group={{ dcld.group }}
+Type=simple
+Restart=always
+RestartSec=1
+ExecStart={{ dcld.path }} start --home {{ dcld.home }}
```

Comment:
`--home` here is a bit different than home directory for a user, as stated in `dcld` help (that is actually comes from tendermint)

>      --home string         directory for config and data (default "<user-home>/.dcl")

I suggest to rely on default setting: take care about user home creation (`/var/lib/dcld`) and allow `dcld` service to use `$HOME/.dcl` as a home directory for the dcl data, so `--home` is not necessary